client/src/actions/search.js

```diff
@@ -31,8 +31,9 @@ export default function searchMonuments(options = {}) {
         const queryString = QueryString.stringify(options);
         dispatch(searchMonumentsPending());
         try {
-            const count = await get('/api/search/count/?', queryString);
-            const monuments = await get('/api/search/?', queryString);
+            const count = await get(`/api/search/monuments/count/?${queryString}`);
```

Comment:
How much faster is the count query? Should we be doing this everywhere we make searching API calls or do you think it's not worth it?

Reply:
We only do the count query because we actually display it on the page. I would not do the query for any other reason (like checking if there's 0, because it's faster to just do the actual search)